src/pages/Profile.js

```diff
@@ -338,4 +455,9 @@ export const addNavPaths = () => {
     component: Profile,
     title: 'Profile'
   })
+  Nav.defPath('fence-callback', {
+    path: '/fence-callback',
```

Comment:
Is this a separate path because we had to register it with them, and we're guarding against future changes?

Also, TOL: I wonder if it would be cleaner to handle the 'handshake' response on a completely separate component, which then redirects to the profile on completion (rather than embedding that process into the profile page itself)

Reply:
Yes, we needed to register the URL with UChicago.

I'd thought about using a separate component (I think you suggested it when looking at the NIH link code). As is, this is more-or-less a port from FireCloud. I like the idea of a separate component and would love to have the experience of performing that refactor. I worry that it would take longer than I'm wanting to spend right now but I will do it if you think it's important to not put it off.

Reply:
Chatted in person. Not refactoring at this time.